### DISCUSSION TITLE
How to set parameters if query length less than 32?

### DISCUSSION THREAD

**Discussion by AmadeusloveIris** [bbuchfink/diamond#564] [Q&A]
Hello, I have a bunch of peptide query which length less than 32. And most of them are less than 15. I find that if I use diamond directly, a lot of query won't get any hit. What should I do? Thx.

**Comment by bbuchfink** [comment]
You may need to change the evalue cutoff for this along with some other internal parameters, see here for some tips: https://github.com/bbuchfink/diamond/discussions/469
